contracts/contracts/SigningLogic.sol

```diff
@@ -1,4 +1,4 @@
-pragma solidity 0.5.0;
+pragma solidity ^0.5.0;
```

Comment:
@SidSethi did we pin this version on purpose? or is this just a typo

Reply:
good catch, i was unsure of that